### ISSUE TITLE
some punctuation breaks the script

### ISSUE THREAD

**Issue by tomhardman0** [tomhardman0/imailer#3]
messages need to be escaped properly.

**Comment by kfatehi** [comment]
yep. i would have fixed this in your module but you did not use `module.exports` so i had to lift the code out and into my own package... but i dealt with the escaping issues in a few ways:

* used es6 backticks for applescript command creation
* escaped quotes in the incoming message in the style required by applescript
* used spawn instead of exec in order to avoid dealing with shell escaping

https://github.com/kfatehi/matrix-appservice-imessage/blob/9c4836cd86aeae5f2bb86edf03b3490d5863184e/send-message.js

if you end up making your package export a function as well (you can do something like this http://stackoverflow.com/questions/13651945/what-is-the-use-of-module-parent-in-node-js-how-can-i-refer-to-the-requireing ) then i will make my package depend on yours.

if not, there's the fix anyway. thanks!
